velox/docs/functions/spark/array.rst

```diff
@@ -15,6 +15,12 @@ Array Functions
 
         SELECT array(1, 2, 3); -- [1,2,3]
 
+.. spark:function:: array_append(array(E), value) -> array(E)
+
+    Add the element at the end of the array passed as first argument. ::
```

Comment:
Perhaps also document the behavior when input array or value is null.

Reply:
Done